settings.py

```diff
@@ -25,4 +25,29 @@
 
 INSTALLED_APPS = [
     "db",
+    "django.contrib.auth",
+    "django.contrib.contenttypes"
+]
+
+AUTH_USER_MODEL = "db.User"
+
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME":
+            "django.contrib.auth.password_validation."
+            "UserAttributeSimilarityValidator"
+    },
+    {
+        "NAME":
+            "django.contrib.auth.password_validation.MinimumLengthValidator"
+    },
+    {
+        "NAME":
+            "django.contrib.auth.password_validation.CommonPasswordValidator"
+    },
+    {
+        "NAME":
+            "django.contrib.auth.password_validation.NumericPasswordValidator"
+    }
```

Comment:
No need to add AUTH_PASSWORD_VALIDATORS here, remove it

Reply:
Thx you ! Removed AUTH_PASSWORD_VALIDATORS from settings.py